praxis.gemspec

```diff
@@ -20,7 +20,7 @@ Gem::Specification.new do |spec|
   spec.bindir = 'bin'
   spec.executables << 'praxis'
 
-  spec.add_dependency 'rack', '~> 1'
+  spec.add_dependency 'rack', '>= 1'
```

Comment:
I don't foresee any rack 3 happening (just released version two now...and rack exists since 2007)...plus it is a low-level framework to major numbers are a big deal to release...so I don't think we should worry about it. IMO at least.